### PR TITLE
[Doc issue][Update-channel-registration-to-bot-resource-in-Azure-portal][2207892]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7e51ab322cab2c9a4ae805da727cabaab04b7600/msteams-platform/bots/how-to/authentication/add-authentication.md) | 83 | 123 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/51f825a5-7587-4ed2-b9c2-a3ce196fc00f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/efa43f10c1255b46abbbd797cb4f736cdd3d45cd/msteams-platform/bots/how-to/authentication/add-authentication.md) | 83 | 123 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b3b4c2fa-9fe7-454c-aed4-9d35b2c8fc6f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a1621e6da5bb4176233f12be3e867a97df43382/msteams-platform/bots/how-to/authentication/add-authentication.md) | 83 | 124 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d15d9c1f-37fa-41a5-8cf5-19d2861fe3b8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/122b2180745c633ae12601710f4db3f0582b50f1/msteams-platform/bots/how-to/authentication/add-authentication.md) | 83 | 124 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1235d6d7-8281-4f22-9f99-d5472f8db759) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — DISMISSED** [review]
Approved

File: msteams-platform/bots/how-to/authentication/add-authentication.md [hunk 1]
```diff
@@ -73,17 +73,79 @@ You use a resource group to create individual resources for the Bot Framework. F
...
+## Create Azure Bot resource registration
 
-The bot channels registration registers your web service as a bot with the Bot Framework, provided you have a Microsoft App Id and App password (client secret).
+The Azure Bot resource registration registers your web service as a bot with the Bot Framework which provides you a Microsoft App Id and App password (client secret).
 
 > [!IMPORTANT]
 > You only need to register your bot if it is not hosted in Azure. If you [created a bot](/azure/bot-service/abs-quickstart?view=azure-bot-service-4.0&viewFallbackFrom=azure-bot-service-3.0&preserve-view=true) through the Azure portal then it is already registered with the service. If you created your bot through the [Bot Framework](https://dev.botframework.com/bots/new) or [Developer Portal](../../../concepts/build-and-test/teams-developer-portal.md) your bot isn't registered in Azure.
 
-[!INCLUDE [bot channels registration steps](~/includes/bots/azure-bot-channels-registration.md)]
+1. Select **Create**.
```
Comment (Wajeed-msft) [inline msteams-platform/bots/how-to/authentication/add-authentication.md:83]: Could you please include more details on which resource to create? 

Can we change it to following?
1. Visit Azure portal and search for **Azure Bot** in create new resource section.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7c3b528f819810f505da86d665d464f5addbc0df/msteams-platform/bots/how-to/authentication/add-authentication.md) | 83 | 125 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6aab132d-3a5b-4778-a993-2cd4ef7bc30d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/authentication/add-authentication.md (hunk 1 above)
Comment (v-npaladugu) [reply msteams-platform/bots/how-to/authentication/add-authentication.md]: Changed it accordingly.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f97efee03af91c3759c34a30c4401e43ba6d8a57/msteams-platform/bots/how-to/authentication/add-authentication.md) | 83 | 125 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6e0f7841-f403-4fcf-ba1c-0d0a21afd581) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/add-authentication.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0f157d26a41883a5e54c35ceecb84e3616184b99/msteams-platform/bots/how-to/authentication/add-authentication.md) | 83 | 125 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5a3f8329-5d29-45ce-abd7-f00a098ef09f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Review by ashishguptaiitb — APPROVED** [review]

**Pull request merged**